frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -490,113 +501,135 @@ const PropertyListView: React.FC = () => {
+          columns={columns}
+          data={data || []}
+          loading={data === undefined}
+          filterable
+          sort={sorting}
+          pageIndex={pageIndex}
+          onRequestData={handleRequestData}
+          pageCount={pageCount}
+          onSortChange={(column: string, direction: SortDirection) => {
+            if (!!direction) {
+              setSorting({ ...sorting, [column]: direction });
+            } else {
+              const data: any = { ...sorting };
+              delete data[column];
+              setSorting(data);
             }
           }}
-        >
-          <Form>
-            <DirtyRowsTracker setDirtyRows={setDirtyRows} />
-            <Table<IProperty>
-              name="propertiesTable"
-              lockPageSize={true}
-              columns={columns}
-              data={data || []}
-              loading={data === undefined}
-              filterable
-              sort={sorting}
-              pageIndex={pageIndex}
-              onRequestData={handleRequestData}
-              pageCount={pageCount}
-              onSortChange={(column: string, direction: SortDirection) => {
-                if (!!direction) {
-                  setSorting({ ...sorting, [column]: direction });
-                } else {
-                  const data: any = { ...sorting };
-                  delete data[column];
-                  setSorting(data);
-                }
-              }}
-              canRowExpand={(val: any) => {
-                if (val.values.propertyTypeId === 0) {
-                  return true;
-                } else {
-                  return false;
-                }
-              }}
-              detailsPanel={{
-                render: val => {
-                  if (expandData[val.id]) {
-                    return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+          canRowExpand={(val: any) => {
+            if (val.values.propertyTypeId === 0) {
+              return true;
+            } else {
+              return false;
+            }
+          }}
+          detailsPanel={{
+            render: val => {
+              if (expandData[val.id]) {
+                return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+              }
+            },
+            icons: {
+              open: <FaFolderOpen color="black" size={20} />,
+              closed: <FaFolder color="black" size={20} />,
+            },
+            checkExpanded: (row, state) => !!state.find(x => checkExpanded(x, row)),
+            onExpand: loadBuildings,
+            getRowId: row => row.id,
+          }}
+          filter={appliedFilter}
+          onFilterChange={values => {
+            setFilter({ ...filter, ...values });
+          }}
+          renderBodyComponent={({ body }) => (
+            <Formik
+              innerRef={tableFormRef as any}
+              initialValues={{ properties: data || [] }}
+              enableReinitialize
```

Comment:
probably will want to make sure that leaving the enableReinitialize isn't refreshing the form if a user is in edit mode for  a while ( a few minutes) also want to make sure that changes in the filter bar affect edit mode in a reasonable way.

Reply:
This seems to be not a problem